File: customview/src/main/java/dinson/customview/utils/IOUtils.java
package dinson.customview.utils;

import java.io.Closeable;
import java.io.IOException;

public class IOUtils {
    /**
     * 关闭IO
     *
     * @param closeables closeables
     */
    public static void close(final Closeable... closeables) {
        if (closeables == null) return;
        for (Closeable closeable : closeables) {
            if (closeable != null) {
                try {
                    closeable.close();
                } catch (IOException e) {
                    e.printStackTrace();
                }
            }
        }
    }
}

File: blingbase/src/main/java/com/dinson/blingbase/utils/IOUtils.kt
package com.dinson.blingbase.utils

import java.io.Closeable
import java.io.IOException

object IOUtils {
    /**
     * 关闭IO
     *
     * @param closeables closeables
     */
    @JvmStatic
    fun close(vararg closeables: Closeable) {
        closeables.forEach {
            try {
                it.close()
            } catch (e: IOException) {
                e.printStackTrace()
            }
        }
    }
}
